host-tools/genlittlefs/CMake/FindBLKID.cmake

```diff
@@ -25,7 +25,7 @@ else (APPLE)
 endif (APPLE)
 FIND_LIBRARY(BLKID_LIBRARIES
         NAMES ${BLKID_NAMES}
-        PATHS /lib64 /lib /usr/lib64 /usr/lib /usr/local/lib64 /usr/local/lib /usr/lib/x86_64-linux-gnu
+        PATHS /lib64 /lib /usr/lib64 /usr/lib /usr/local/lib64 /usr/local/lib /usr/lib/x86_64-linux-gnu /usr/lib/aarch64-linux-gnu
```

Comment:
is this change connected with the issue?

Reply:
No, but I decided I would add it here as it's so minor that there is no need to create separate PR for it. It enables support for Aarch64 Linux (as I pointed in the PR description).